rails/client/webpack.config.js

```diff
@@ -17,8 +17,7 @@ const ManifestPlugin = require('webpack-manifest-plugin');
 const webpackConfigLoader = require('react-on-rails/webpackConfigLoader');
 
 const configPath = resolve('..', 'config');
-const { devBuild, output, webpackOutputPath, webpackPublicOutputDir } =
-  webpackConfigLoader(configPath);
```

Comment:
`'devBuild' is assigned a value but never used                no-unused-vars
 'webpackOutputPath' is assigned a value but never used       no-unused-vars
 'webpackPublicOutputDir' is assigned a value but never used  no-unused-vars`